lib/libmbedtls/core/tee_lmd_provider.c

```diff
@@ -490,55 +494,90 @@ TEE_Result crypto_hash_final(void *ctx, uint32_t algo, uint8_t *digest,
  ******************************************************************************/
 
 #if defined(_CFG_CRYPTO_WITH_ACIPHER)
-struct bignum *crypto_bignum_allocate(size_t size_bits __unused)
+#define ciL		(sizeof(mbedtls_mpi_uint))	/* chars in limb  */
+#define biL		(ciL << 3)			/* bits  in limb  */
+#define BITS_TO_LIMBS(i) ((i) / biL + ((i) % biL != 0))
+
+#define LMD_MAX_BITS_PER_VARIABLE	(4096)
```

Comment:
This should be based on `CFG_CORE_BIGNUM_MAX_BITS`, or even better use `CFG_CORE_BIGNUM_MAX_BITS` instead.

Reply:
Actually, we do not use `bn_alloc_max()` anymore in this PR. So this macro can be deleted.
But I think we should check the `size_bits` with the MAX macro before allocate memory in `crypto_bignum_allocate()`. What do you think?

Reply:
Agree